source/common/network/utility.cc

```diff
@@ -328,8 +328,9 @@ Address::InstanceConstSharedPtr Utility::getAddressWithPort(const Address::Insta
     return std::make_shared<Address::Ipv4Instance>(address.ip()->addressAsString(), port);
   case Address::IpVersion::v6:
     return std::make_shared<Address::Ipv6Instance>(address.ip()->addressAsString(), port);
+  default:
+    return nullptr;
   }
-  PANIC("not handled");
 }
```

Comment:
Why was this changed? What case causes this to be hit now?

Reply:
There was [some concern](https://github.com/howardjohn/envoy/commit/53a23d5a3df705338c776eb6c500d793218a0367#r74719909) that we might panic in the original PR.  I figured it's better to allow the callers to handle a nullptr than to just give up.  Fine with reverting this if we don't care.

Reply:
Let's just revert for now.